docker/sysctl-setter/Dockerfile

```diff
@@ -12,6 +12,14 @@ while true; do\n\
   if [ "$SVAL" -ne 1 ]; then\n\
     sysctl -w vm.swappiness=1\n\
   fi\n\
+  FVAL=$(sysctl -n fs.file-max)\n\
+  if [ "$FVAL" -ne 3000000 ]; then\n\
+    sysctl -w fs.file-max=3000000\n\
+  fi\n\
+  OVAL=$(sysctl -n fs.nr_open)\n\
+  if [ "$OVAL" -ne 3000000 ]; then\n\
```

Comment:
Can we change this (and the map map count section) to be `-le` instead of `-ne`? That way if someone already has a higher val we don't lower it